build/processLib.js

```diff
@@ -0,0 +1,63 @@
+// Porcess generated lib files.
+// Like adding js extension in the import statement.
+
+const { transformImport } = require('./transformImport');
+const globby = require('globby');
+const path = require('path');
+const fs = require('fs');
+const chalk = require('chalk');
+const rollup = require('rollup');
+const nodeResolve = require('@rollup/plugin-node-resolve').default;
+
+function addJsExtension(moduleName) {
+    // Ignore 'tslib'
+    if (!(moduleName.startsWith('.'))) {
+        return moduleName;
+    }
+    if (moduleName.endsWith('.ts')) {
+        // Replace ts with js
+        return moduleName.replace(/\.ts$/, '.js');
+    }
+    else if (moduleName.endsWith('.js')) {
+        return moduleName;
+    }
+    else {
+        return moduleName + '.js'
+    }
+}
+
+async function transform() {
+    const libFiles = await globby(path.join(__dirname, '../lib/**/*.js'));
```

Comment:
I can remember I mentioned somewhere a compatibility issue of `globby` in Windows.

It's suggested to use the `cwd` option to specify the scan directory, or it can't match any files in Windows.


```js
const libFiles = await globby([
    '**/*.js'
], {
    cwd: path.join(__dirname, '../lib'),
    absolute: true
})
```

Reply:
Thanks for the remind! I did some search and found it seems to be related to this issue https://github.com/sindresorhus/globby/issues/179#issuecomment-990398774 . Just fixed as you suggested.